File: rgb-to-hex-conversion/rgb-to-hex-conversion.js
function rgb(r, g, b){
        console.log(r,g,b)
  var s = [r,g,b].map(a=>Math.max(0,Math.min(255,a))!=0?Math.max(0,Math.min(255,a)).toString(16):"00")
  return s.join("").toUpperCase()
}

File: rgb-to-hex-conversion/rgb-to-hex-conversion.py
def rgb(r, g, b):
    s = ""
    for i in [r,g,b]:
        if 0<i<=255:
            s += hex(i)[2:] if i>10 else "0"+hex(i)[2:]
        elif i>255:
            s += "FF"
        else:
            s += "00"
    return s.upper()